Makefile

```
@@ -0,0 +1,8 @@
+.PHONY: test
+test:
+	go test ./... -v
```

from Test Security Reviewer
テストの実行オプションを改善します。タイムアウト設定、並列実行、レースディテクタを追加します。

```suggestion
	go test ./... -v -timeout 30s -race -parallel 4
```